SECURITY.md

```diff
@@ -0,0 +1,30 @@
+# Security Policy
+
+SigLens takes the security of our software applications very seriously. If you believe you have found a security issue in our applications, please report it to us as described below.
+
+## Supported Versions
+We always recommend using the latest version of SigLens to ensure you get all security updates.
+
+## Reporting a Vulnerability
+
+**Do not report vulnerabilities using public GitHub issues**. 
+
+If you have found a security issue within SigLens, please email <info@sigscalr.io> with a detailed account of the issue. We'll try and fix the problem as soon as possible.
+
+Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
+
+  * Type of issue (e.g. segmentation violation, buffer overflow, SQL injection, cross-site scripting, etc.)
+  * Full paths of source file(s) related to the manifestation of the issue
+  * The location of the affected source code (tag/branch/commit or direct URL)
```

Comment:
nit: changing this line to `The code version (tag/branch/commit or direct URL)` might be a little more clear, unless you mean something else